src/capitalize_name.rb

```diff
@@ -0,0 +1,14 @@
+# capitalize class
+require_relative 'nameable'
+
+class Capitalize < Nameable
```

Comment:
- [ ] The capitalizer class should inherit from Decorator, not Nameable. Kindly fix that.